includes/abstracts/class-llms-rest-users-controller.php

```diff
@@ -593,7 +595,9 @@ protected function prepare_object_for_response( $object, $request ) {
 		unset( $map['user_pass'] );
 
 		foreach ( $map as $db_key => $schema_key ) {
-			$prepared[ $schema_key ] = $object->get( $db_key );
+			if ( in_array( $schema_key, $fields, true ) ) {
```

Comment:
this is already merged in dev, but this PR is against trunk

Reply:
woops